### PR TITLE
touchevents: Use mq in touch event detection.

### PR DESCRIPTION
**Pull request by emilio**
testStyles + offsetTop is unnecessarily slow if the browser supports matchMedia.

### CODE REVIEW

**Comment by emilio** [comment]
The Travis and AppVeyor failures look unrelated (and the Travis + node 6 run is green), but let me know if they're not.

File: feature-detects/mediaquery/hovermq.js [hunk 1]
```diff
@@ -12,5 +12,5 @@
 Detect support for Hover based media queries
 */
 define(['Modernizr', 'addTest', 'mq'], function(Modernizr, addTest, mq) {
-  Modernizr.addTest('hovermq', mq(('(hover)')));
```
Comment (patrickkettner) [inline feature-detects/mediaquery/hovermq.js]: why this change?

Comment (emilio) [reply feature-detects/mediaquery/hovermq.js]: I just saw the extra parentheses while I was going through similar patterns. I can remove it if needed, but I thought a bit of cleanup never hurts. Let me know if you want to remove that commit from this PR and submit it separately.

Comment (patrickkettner) [reply feature-detects/mediaquery/hovermq.js]: no we're good, just making sure there wasn't an issue

**Comment by rejas** [comment]
@emilio @patrickkettner the PR https://github.com/Modernizr/Modernizr/pull/2294 would fix the Travis and AppVeyor failures

**Comment by bholley** [comment]
@emilio what's the status here?

**Comment by bholley** [comment]
Ok, great! I just remeasured, and confirmed that this fix still saves ~15ms on cnn pageload.

@patrickkettner, any chance y'all could do another dot release (similar to #2291) so that we can get this into CNN's pipeline?

**Comment by rejas** [comment]
would love to see a v3.7 out the door too, alas we are still behind with the release notes for 3.6 :-(
